prime-dotnet-webapi/Controllers/EmailsController.cs

```diff
@@ -52,5 +56,38 @@ public async Task<ActionResult> SendEnrolleeRenewalEmails()
 
             return NoContent();
         }
+
+        // Email Templates
+
+        // GET: api/emails/management/templates
+        /// <summary>
+        /// Get email templates
+        /// </summary>
+        [HttpGet("management/templates", Name = nameof(GetEmailTemplates))]
+        [Authorize(Roles = Roles.ViewEnrollee + "," + Roles.ViewSite)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ApiResultResponse<EmailTemplateListViewModel>), StatusCodes.Status200OK)]
```

Comment:
should be `typeof(ApiResultResponse<IEnumerable<EmailTemplateListViewModel>>)`C#